Toggl.PrimeRadiant.Realm/Database.cs

```diff
@@ -98,6 +99,18 @@ private RealmConfiguration createRealmConfiguration()
                     {
                         // nothing needs explicit updating when updating from schema 4 up to 6
                     }
+
+                    if (oldSchemaVersion < 7)
+                    {
+                        var newWorkspaces = migration.NewRealm.All<RealmWorkspace>();
+                        var oldWorkspaces = migration.OldRealm.All("RealmWorkspace");
```

Comment:
why not a `nameof()`?
```suggestion
                        var oldWorkspaces = migration.OldRealm.All(nameof(RealmWorkspace));
```

Reply:
The name of the class might change in the future (let’s not do that though!) and the table will still be accessible only under the old name.

Reply:
Don't you think it would be better to change it to specific name when that thing happens?
I am sorry to be a pain in the ass for such a simple PR...